### PR TITLE
Fix some errors phpstan

### PR DESCRIPTION
**Pull request by okom3pom**

### CODE REVIEW

File: src/Commands/Customers/CustomersGroups.php [hunk 1]
```diff
@@ -208,7 +208,7 @@ private function groupCustomersTransfer(
         }
 
         // Delete the form group
-        if (self::ACTION_DELETE_FROM_GROUP == (int) $actionAfter && $hasError != 0) {
+        if (self::ACTION_DELETE_FROM_GROUP == (int) $actionAfter && 0 == $hasError) {
```
Comment (SebSept) [inline src/Commands/Customers/CustomersGroups.php:211]: you changed `==` with `!=` it's not a phpstan fix...

Comment (okom3pom) [reply src/Commands/Customers/CustomersGroups.php:211]: At first it was `!$hasError`

The error was : 

When I corrected it I was wrong

Comment (SebSept) [reply src/Commands/Customers/CustomersGroups.php:211]: right.
sorry.

**Review by SebSept — APPROVED** [review]
lgtm

**Review by jf-viguier — APPROVED** [review]

**Comment by jf-viguier** [comment]
merci Thomas

**Pull request merged**